src/services/location.js

```diff
@@ -0,0 +1,18 @@
+export const GetLocation = () => {
+    return new Promise((resolve, reject) => {
+        if (navigator.geolocation) {
+            navigator.geolocation.getCurrentPosition(
+                (position) => {
+                    const { latitude, longitude } = position.coords;
+                    resolve({ latitude, longitude });
+                },
+                (error) => {
+                    console.error(error);
+                    reject(new Error("Failed to retrieve location."));
+                }
+            );
+        }
+    });
+};
+
+export default GetLocation;
```

Comment:
If you're exporting as a named export (line 1) there's no need to export as a default.

https://www.geeksforgeeks.org/difference-between-default-named-exports-in-javascript/